views/js/helper/status.js

```diff
@@ -37,7 +37,9 @@ define(['lodash', 'i18n'], function(_, __){
                 report : true,
                 print : __('not finished'),
                 reactivate : __('not terminated'),
-                time : true,
+                time : function(delivery) {
```

Comment:
@andrey-ponamarev Here is an example (and the reason of the changes) of how can be permissions be dynamically managed now with a function.